### PR TITLE
Bug Support + Staff Link + Login Feature

### PR DESCRIPTION
**Pull request by TheGamerzs**

### CODE REVIEW

**Comment by Timeraa** [comment]
@TheDropX

**Comment by TheDropX** [comment]
This looks good, but check this first.
https://github.com/PreMiD/API/pull/131

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,7 @@ static/assets
 dist
 .vscode/settings.json
 src/langs
-.nuxt
\ No newline at end of file
+.nuxt
+nuxt.config.js
```
Comment (eggsywashere) [inline .gitignore:8]: nononononono why 🚫 🙅

File: src/components/Navigation.vue [hunk 2]
```diff
@@ -37,39 +38,61 @@
...
+							<span id="tag">#{{ $auth.user.discriminator }}</span>
+						</span>
+						</span>
+					</nuxt-link>
+
+					<nuxt-link v-if="$auth.loggedIn"
+						:to="''"
+						ref="userLinks">
+						<span id="user-link">
+							<a id="userLinks" v-if="$isStaff === false" @click="redirect('/bug')" v-t="'header.rab'">{{ $t("header.rab") }}</a>
```
Comment (Timeraa) [inline src/components/Navigation.vue:60]: $isStaff > isStaff

File: src/components/Navigation.vue [hunk 3]
```diff
@@ -340,26 +387,45 @@
...
+							"673683121971134505", //Head of Presence Verifying
+							"691384256672563332", //Community Representative
+							"514546359865442304", //Moderator
+							"526734093560315925", //Junior Moderator
+							"566417964820070421", //Support Agent
+							"630445337143935009" //Presence Verifier
+						];
+
+						if (staffRoles.indexOf(data.roleId) !== -1) this.$isStaff = true;
+						else this.$isStaff = false;
```
Comment (Timeraa) [inline src/components/Navigation.vue:424]: $isStaff > isStaff

File: src/components/Navigation.vue [hunk 4]
```diff
@@ -37,39 +38,61 @@
...
+						<span id="user-link">
+							<a id="userLinks" v-if="$isStaff === false" @click="redirect('/bug')" v-t="'header.rab'">{{ $t("header.rab") }}</a>
+							<a id="userLinks" v-else @click="redirect('/staff')" v-t="'header.staff'">{{ $t("header.staff") }}</a>
+							<a id="userLinks" @click="redirect('/logout')" v-t="'header.logout'">{{ $t("header.logout") }}</a>
+						</span>
+					</nuxt-link>
+
+
+					<nuxt-link v-if="!this.$auth.loggedIn"
+						:key="login"
```
Comment (Timeraa) [inline src/components/Navigation.vue:68]: :key > key="login"

**Comment by TheGamerzs** [comment]
Needs Collection: 'bugs' and 'bugUsers'

File: src/components/Navigation.vue (hunk 3 above)
Comment (TheGamerzs) [reply src/components/Navigation.vue]: Updated

File: src/components/Navigation.vue (hunk 2 above)
Comment (TheGamerzs) [reply src/components/Navigation.vue]: Updated

File: src/components/Navigation.vue (hunk 4 above)
Comment (TheGamerzs) [reply src/components/Navigation.vue]: Updated

File: .gitignore (hunk 1 above)
Comment (TheGamerzs) [reply .gitignore]: Updated

**Pull request merged**